app/modules/parsing/tasks/cache_cleanup_tasks.py

```
@@ -0,0 +1,62 @@
+
+@celery_app.task(bind=True, name="cache_cleanup.get_stats")
+def get_cache_cleanup_stats(self: Task):
+    """Task to get cache cleanup statistics"""
+    try:
+        db = next(get_db())
+        cleanup_service = CacheCleanupService(db)
+
+        stats = cleanup_service.get_cleanup_stats()
+        logger.info(f"Cache stats: {stats}")
+
+        return {"stats": stats, "status": "success"}
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)